api/algolia-sync.js

```
@@ -0,0 +1,161 @@
+
+    const missingEnvVars = requiredEnvVars.filter(varName => !process.env[varName]);
+    if (missingEnvVars.length > 0) {
+        console.error('Missing Environment Variables:', missingEnvVars);
+        return res.status(500).json({
+            message: 'Server Configuration Error',
+            missingVars: missingEnvVars
+        });
+    }
+
+    // Initialize clients
+    let client, index, ghost;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)